app/gateway-manager/data-plane-reference.md

```diff
@@ -13,24 +12,213 @@ works_on:
+
+         For example, if the external IP is `35.233.198.16` and your route is `/mock`, access your service at:
+         ```
+         http://35.233.198.16:80/mock
+         ```
+  - q: Can I choose a specific {{site.base_gateway}} version when using Quickstart scripts in Gateway Manager?
+    a: |
+      Yes. Gateway Manager allows you to select the {{site.base_gateway}} version for your Quickstart scripts—except when using cloud provider scripts for AWS, Azure, and GCP.
+
+  - q: Can I SSH directly into Konnect Data Plane nodes?
+    a: |
+      No. Direct SSH access is not possible because the SSH keys are randomly generated and not exposed. 
+      To access nodes, use the cloud provider’s tools:
+      * [AWS EC2 Instance Connect](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ec2-instance-connect-methods.html)
+      * [Azure Cloud Shell](https://learn.microsoft.com/azure/cloud-shell/overview)
+      * [Google Cloud SSH](https://cloud.google.com/compute/docs/instances/ssh)
+
 
 related_resources:
-  - text: Konnect Advanced Analytics
-    url: /advanced-analytics/
+  - text: Gateway Manager
+    url: /gateway-manager/reference/
   - text: Dedicated Cloud Gateways
     url: /dedicated-cloud-gateways/
   - text: Serverless Gateways
     url: /serverless-gateways/
 ---
 
-@TODO
 
-pull content from 
-https://docs.konghq.com/konnect/gateway-manager/data-plane-nodes/
-https://docs.konghq.com/konnect/gateway-manager/data-plane-nodes/upgrade/
-https://docs.konghq.com/konnect/gateway-manager/data-plane-nodes/verify-node/
-https://docs.konghq.com/konnect/gateway-manager/data-plane-nodes/renew-certificates/
-https://docs.konghq.com/konnect/gateway-manager/data-plane-nodes/parameter-reference/
-https://docs.konghq.com/konnect/gateway-manager/data-plane-nodes/custom-dp-labels/
\ No newline at end of file
+A Data Plane node is a single instance of {{site.base_gateway}} that acts as a proxy and serves traffic.
+In {{site.konnect_short_name}}, Data Plane nodes are managed by [Control Planes](/gateway-manager/control-plane-groups/). Control planes manage and store configurations in {{site.konnect_short_name}}, and Data Plane nodes are configured according to the configuration distributed by the Control Plane.
+
+{{site.konnect_short_name}} provides Data Plane node installation scripts for various platforms. 
+These Data Plane nodes are configured to run in your {{site.konnect_short_name}} environment. Alternatively, {{site.konnect_short_name}} offers fully-managed Data Planes through [Dedicated Cloud Gateways](/dedicated-cloud-gateways).
+
+## Supported installation options
+
+{{site.konnect_short_name}} supports the following installation options:
+
+{% table %}
+columns:
+  - title: Setup Type
+    key: type
+  - title: Platforms
+    key: platforms
+rows:
+  - type: Standard setup
+    platforms: macOS (ARM), macOS (Intel), Windows, Linux (Docker)
+  - type: Advanced setup
+    platforms: Linux, Kubernetes, AWS, Azure, Google Cloud
```

Comment:
```suggestion
    platforms: Linux, Kubernetes
```
Those might be supported but not via quickstart scripts. I imagine you'd have to figure those out yourself.